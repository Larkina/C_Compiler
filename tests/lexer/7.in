0.2

0.36e+1

666e-5

123e5