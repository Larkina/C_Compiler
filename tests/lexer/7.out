Token{value=0.2, text=n10, type=INT}
Token{value=3.6, text=n10, type=INT}
Token{value=0.00666, text=n10, type=INT}
Token{value=1.23E7, text=n10, type=INT}
